src/Microsoft.Extensions.Configuration.FileProviderExtensions/ConfigurationRootExtensions.cs

```diff
@@ -20,8 +21,13 @@ public static IConfigurationRoot ReloadOnChanged(this IConfigurationRoot config,
                 throw new ArgumentNullException(nameof(filename));
             }
 #if NET451
-            var basePath = AppDomain.CurrentDomain.GetData("APP_CONTEXT_BASE_DIRECTORY") as string ??
-                AppDomain.CurrentDomain.BaseDirectory ?? string.Empty;
+            var basePath =
```

Comment:
I missed that while reading reading the signature of the method. I don't think there is anything else we should do as part of this change. Currently we require users to provide the location and name of the file again to setup auto-reloading of the configuration if the file changes. I guess they will have to combine the base path on their own if it is not the default. Sorry for the waste of time :disappointed: